app/filtering.js

```diff
@@ -571,7 +581,10 @@ module.exports.isResourceEnabled = (resourceName, url) => {
   }
 
   if ((resourceName === appConfig.resourceNames.ADBLOCK ||
-       resourceName === appConfig.resourceNames.TRACKING_PROTECTION)) {
+      appConfig[resourceName] &&
```

Comment:
Yep it was probably there because at one point they mattered and then after a bugfix they were left behind. Fixed as part of:
ecc67f28714ec923d59d639f79415364712ad214